ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx

```diff
@@ -183,10 +188,12 @@ export function TimeSeriesChartPanel(props: TimeSeriesChartProps) {
         // Used for repeating colors in Categorical palette
         seriesIndex++;
 
-        const seriesId = timeSeries.name + seriesIndex;
+        // We add a unique id for the chart to disambiguate items across charts
+        // when there are multiple on the page.
```

Comment:
If we don't do this, when `connect` is enabled, all the items with the same ids (a combo of the label + the index) would select, which we decided we don't want to do right now.